changelog.d/20240604_110454_fnareoh_refacto_censor_lines.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+The display of patch start of lines symbols has been made more uniform.
+It is always displayed not just in case of multiline secrets.
```

Comment:
I think this entry can be more concise, here is an attempt at shortening it (also, turn it into a list item, otherwise scriv is not going to be happy)

```suggestion
- Patch symbols at the start of lines are now always displayed, even for single line secrets.
```